logging/tests/system.py

```diff
@@ -108,6 +108,7 @@ class TestLogging(unittest.TestCase):
             'precipitation': False,
         },
     }
+    TYPE_FILTER = 'protoPayload.@type = "{}"'
```

Comment:
@waprin I already ran this so I know it "works on my machine". But is this the correct syntax to be using for a filter?